src/components/cards/LearnCard.tsx

```diff
@@ -0,0 +1,126 @@
+import { Box, Column, Columns, Inline, Stack, Text } from '@/design-system';
+import React, { useCallback } from 'react';
+import { GenericCard } from './GenericCard';
+import { getLearnCardColorway } from './utils/constants';
+import { LearnCardDetails } from './utils/types';
+import { IconOrb } from './reusables/IconOrb';
+import { useNavigation } from '@/navigation';
+import Routes from '@/navigation/routesNames';
+import * as i18n from '@/languages';
+import { useTheme } from '@/theme';
+import { useRoute } from '@react-navigation/native';
+import { analyticsV2 } from '@/analytics';
+
+export const LEARN_CARD_HEIGHT = 184;
+
+type LearnCardProps = {
+  cardDetails: LearnCardDetails;
+  type: 'square' | 'stretch';
```

Comment:
Same as `GenericCard`? Should probably be shared, eh? Should still live with GenericCard though imo.

Reply:
sorry could you elaborate? idk what you mean by shared. but yeah, this prop is the same as the `GenericCard` prop of type `CardType`

Reply:
oh i see the problem